src/include/sof/dma.h

```diff
@@ -71,6 +71,7 @@
 #define DMA_DEV_HDA		BIT(1) /**< connectable to HD/A link */
 #define DMA_DEV_SSP		BIT(2) /**< connectable to SSP fifo */
 #define DMA_DEV_DMIC		BIT(3) /**< connectable to DMIC fifo */
+#define DMA_DEV_SSI		BIT(4) /**< connectable to SSI / SPI fifo */
```

Comment:
SSI is a Freescale/NXP interface, we'd want to have something less ambiguous here. This is really Intel-specific